<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ssmr::ObisValueMappingWidget</class>
 <widget class="QWidget" name="Ssmr::ObisValueMappingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="3,3,1,2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtDescription">
     <property name="toolTip">
      <string>The description for the value received through the given OBIS number.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="edtObisValue">
     <property name="toolTip">
      <string>The OBIS number to monitor.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="edtUnit"/>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="edtInterval"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
